modules/cardano-explorer.nix

```diff
@@ -1,3 +1,10 @@
+let
+
+  explorer' = (import ./../default.nix {}).cardano-sl-explorer-static;
+  explorer  = builtins.trace "using cardano-sl-explorer ${explorer'}" explorer';
```

Comment:
`trace` shouldn't be used to pollute the deployment log. The source of information what is going to be deployed is at various places, like closure of the machine or `cardano-sl-explorer-src.json`. But we should come up with why we need this information printed and then what tooling to create around it.